Test/warning_test.py

```diff
@@ -542,4 +542,8 @@
        [0, 0]])
 >>> cm4
 pycm.ConfusionMatrix(classes: [1, 2])
+>>> with warns(RuntimeWarning):
```

Comment:
Would you add `match='The curve is undefined at some points.'`.

Reply:
You are right!
But the `match` option sometimes doesn't work correctly. I think we should work on it in another PR.